shell/platform/tizen/flutter_tizen_engine.h

```diff
@@ -24,6 +24,11 @@
 #include "flutter/shell/platform/tizen/public/flutter_tizen.h"
 #include "flutter/shell/platform/tizen/tizen_event_loop.h"
 #include "flutter/shell/platform/tizen/tizen_renderer.h"
+
+#ifndef __X64_SHELL__
+#include "flutter/shell/platform/tizen/channels/app_control_channel.h"
+#endif
+
```

Comment:
Could you move these lines back to line 14 and remove vertical whitespaces, for consistency with `#ifdef TIZEN_RENDERER_EVAS_GL` at line 32-37?